File: A/Hardware.java
public class Hardware extends Product{
    private String brand;
    private String model;

    public Hardware(){
        this.brand = "";
        this.model = "";
    }

    public void setBrand(String b){
        this.brand = b;
    }

    public String getBrand(){
        return brand;
    }

    public void setModel(String m){
        this.model = m;
    }

    public String getModel(){
        return model;
    }
}

File: B/Python/Hardware.py
from Product import Product

class Hardware(Product):
    __brand = ""
    __model = ""
    
    def __init__(self):
        self.__brand = ""
        self.__model = ""
    
    def setBrand(self, b):
        self.__brand = b
    
    def getBrand(self):
        return self.__brand
    
    def setModel(self, m):
        self.__model = m
    
    def getModel(self):
        return self.__model